docs/docs/ref-01-top-level-api.md

```diff
@@ -36,6 +36,19 @@ Create and return a new `ReactElement` of the given type. The type argument can
 html tag name string (eg. 'div', 'span', etc), or a `ReactClass` (created via `React.createClass`).
 
 
+### React.cloneElement
+
+```
+ReactElement cloneElement(
+  ReactElement element,
+  [object props],
+  [children ...]
```

Comment:
Meh, I prefer incremental correctness over consistency, but sure.  Looks good to me.